index.js

```diff
@@ -96,15 +96,24 @@ const drawCommand = (message) => {
           // console.log(JSON.stringify(response.data));
 
           var ref = db.ref("users");
-          ref.update({
-            [message.author.id]: {
-              lastDrawTime: Date.now(),
-              name: message.author.username,
+          const userRef = ref.child(message.author.id);
+          userRef.update({
+            lastDrawTime: Date.now(),
+            name: message.author.username,
+          });
+
+          const inventoryRef = ref.child(message.author.id + "/inventory");
```

Comment:
this works! another way to think about it is:
```js
const inventoryRef = userRef.child("inventory");
```

`.child` takes a `ref` and creates a new `ref`, so you can keep chaining them like this.